myokit/formats/sbml/_exporter.py

```diff
@@ -0,0 +1,54 @@
+#
+# Exports to SBML.
+#
+# This file is part of Myokit.
+# See http://myokit.org for copyright, sharing, and licensing details.
+#
+import myokit
+from myokit.formats.sbml._writer import write_file
```

Comment:
Can `write_file` be imported into `myokit.formats.sbml` so that it's all possible publicly as well? (the exporter is always a bit of a shortcut)

Reply:
yup, done